src/pages/home/sidebar/SidebarScreen/getShouldDisplayInDefaultIconColor.ts

```diff
@@ -0,0 +1,13 @@
+import getPlatform from '@libs/getPlatform';
+import CONST from '@src/CONST';
+
+/**
+ *
+ * @returns boolean true for native platforms (Android & iOS)
+ */
+function getShouldDisplayInDefaultIconColor(): boolean {
+    const platform = getPlatform();
+    return platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID;
```

Comment:
lets create `getShouldDisplayInDefaultIconColor` folder inside /libs and return true from `index.ts` return false from `index.native.ts`.

Reply:
Done!